benchmarking/frameworks/caffe2/caffe2.py

```diff
@@ -342,7 +342,10 @@ def _processDelayData(self, data):
             for k, v in d.items():
                 for kk, vv in v.items():
                     key = k + " " + kk
-                    details[key]["values"].append(float(vv["value"]))
+                    if "info_string" in vv:
+                        details[key]["info_string"] = vv["info_string"]
```

Comment:
Here there are multiple values, may we assert if the values don't match?

Reply:
I'm sorry, I'm not sure I understand. What multiple values are you concerned with?

Reply:
Sorry just following up on this comment. Can you please clarify which multiple values you are referring to and which assertions should be made?